Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/input/model/InputMappingDoubleSetting.java

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
What's going on with this class? Do we lose minimal accuracy when casting down? Why wouldn't you make an `AbstractDoubleSetting` interface?

Reply:
I just figured that it would be kind of a headache to make all the settings UI code support double settings in addition to the float settings that are currently supported. The reduction in precision is more or less meaningless for what we're doing here (we're normally just storing integers in these settings, and occasionally maybe one decimal or so).

Reply:
Fair enough, I won't worry about it too much